stl/inc/valarray

```diff
@@ -1372,6 +1372,11 @@ public:
     _NODISCARD size_t stride() const noexcept /* strengthened */ {
         return _Stride;
     }
+#if _HAS_CXX200
+    friend bool operator==(const slice& x, const slice& y) {
+        return x.start() == y.start() && x.size() == y.size() && x.stride() == y.stride();
+    }
+#endif

```

Comment:
Nitpick: This `#endif` is missing a comment. We conventionally comment `#else` and `#endif` directives to help indicate the boundaries of preprocessor regions; this is especially helpful as the regions grow (when it's easy to forget exactly what condition we're in) or when preprocessor logic is nested.

We sometimes skip comments when an `#if`/`#endif` pair spans a single line and isn't nested, but this is a 3-editor-line function.